script/cadence/transactions/proxy-to-flow-transfer.cdc

```diff
@@ -1,6 +1,6 @@
 import FlowColdStorageProxy from 0xProxy
-import FlowToken from 0x0ae53cb6e3f42a79
-import FungibleToken from 0xee82856bf20e2aa6
+import "FlowToken"
+import "FungibleToken"
 
 transaction(sender: Address, receiver: Address, amount: UFix64) {
 
```

Comment:
I'm not able to make a suggestion for the value below, but this needs to be updated to

```cadence
let xfer: @{FungibleToken.Vault}
```